lib/intro_slider.dart

```diff
@@ -9,6 +9,8 @@ import 'list_rtl_language.dart';
 import 'scrollbar_behavior_enum.dart';
 import 'slide_object.dart';
 
+typedef BottomLayoutBuilder = Widget Function(Widget dots);
```

Comment:
Can you give a code example for this Widget, I haven't got what you would like to do here.

Reply:
Sure.
Here comes my case.

```dart
IntroSlider(
    slides: slides,
    colorDot: ResColors.grey400,
    colorActiveDot: ResColors.blue,
    renderCustomBottom: renderBottom,
)
```

And the `renderBottom` function.
```dart
 Widget renderBottom(Widget dots) {
    return Positioned(
      bottom: 10,
      left: 10,
      right: 10,
      child: SizedBox(
        height: 130,
        child: Column(
          crossAxisAlignment: CrossAxisAlignment.center,
          children: [
            Padding(
              padding: const EdgeInsets.symmetric(vertical: 20),
              child: Row(children: [dots]),
            ),
            TextButton(
              onPressed: () => onFinish(context),
              style: TextButton.styleFrom(
                padding: EdgeInsets.zero,
                minimumSize: Size(180, 40),
                backgroundColor: ResColors.blue,
                shape: RoundedRectangleBorder(
                  borderRadius: BorderRadius.circular(40),
                ),
              ),
              child: I18nText(
                'page.splash.start',
                style: TextStyle(color: ResColors.realWhite, fontSize: 16),
                testKey: Key(TestKeys.splashStart),
              ),
            ),
          ],
        ),
      ),
    );
  }
```

Reply:
Some users would like to keep splash screen simple, so I just need dot indicators and a button to skip or finish the splash screen.

Reply:
![simulator_screenshot_CD127515-97C2-4856-BCC7-CADCF19CE3B8](https://user-images.githubusercontent.com/13448248/122643862-c3ecf600-d144-11eb-94e1-ae034cd39b08.png)